automated-actions/AWS_RISK_CREDENTIALS_EXPOSED/README.md

```diff
@@ -0,0 +1,173 @@
+    * Delete the exposed IAM Access Key Pair
+    * Summarize recent API activity for the user from CloudTrail
+    * Send security notification message containing summary to `SecurityNotificationTopic` SNS Topic.
+
+## Running the Example
+ 
+#### Option 1: Launch the CloudFormation Template in US East - N. Virginia (us-east-1) 
+The backend infrastructure can be deployed in US East - N. Virginia (us-east-1) using the provided CloudFormation template.
+Click **Launch Stack** to launch the template in the US East - N. Virginia (us-east-1) region in your account:
+
+[![Launch Stack into N. Virginia with CloudFormation](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/images/cloudformation-launch-stack-button.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=AWSHealthCredsExposed&templateURL=https://s3.amazonaws.com/aws-health-tools-projects/AWS-RISK-CREDENTIALS-EXPOSED/risk_credentials_exposed.output.yaml)
+
+(In the last page of the wizard, make sure to:
+
+1. Click the checkboxes to give AWS CloudFormation permission to **"I acknowledge that AWS CloudFormation might create IAM resources"** and **"I acknowledge that AWS CloudFormation might create IAM resources with custom names"**
+1. Follow the instructions to **"Create Change Set"** 
+1. Click **"Execute"**
+)
+
+#### Option 2: Launch the CloudFormation Template Manually 
+
+If you would like to deploy the template manually, you need a S3 bucket in the target region, and then package the Lambda functions into that S3 bucket by using the `aws cloudformation package` utility.
+
+Set environment variables for later commands to use:
+
+```bash
+S3BUCKET=[REPLACE_WITH_YOUR_BUCKET]
+REGION=[REPLACE_WITH_YOUR_REGION]
+STACKNAME=[REPLACE_WITH_DESIRED_NAME]
+```
+
+Then go to the `cloudformation` folder and use the `aws cloudformation package` utility
+
+```bash
+cd cloudformation
+
+aws cloudformation package --region $REGION --s3-bucket $S3BUCKET --template risk_credentials_exposed.serverless.yaml --output-template-file risk_credentials_exposed.output.yaml
+```
+Last, deploy the stack with the resulting yaml (`risk_credentials_exposed.output.yaml`) through the CloudFormation Console or command line:
+
+```bash
+aws cloudformation deploy --region $REGION --template-file risk_credentials_exposed.output.yaml --stack-name $STACKNAME --capabilities CAPABILITY_NAMED_IAM
+```
+
+## Testing the Example
+To test the example without exposing an IAM Access Key to a public repository you can simulate the workflow by executing the `ExposedKey` state machine with a set of test json for the event. To do this follow the steps detailed below.
+
+Ensure you have at least one E-mail address [subscribed](http://docs.aws.amazon.com/sns/latest/dg/SubscribeTopic.html) to the `SecurityNotificationTopic` created by the template to receive the notification.
```

Comment:
Fixed in this commit:

https://github.com/aws/aws-health-tools/pull/26/commits/8e5e53f49aa6001c0aa5a2b229b24b2a47bcd1f1